app/ide-desktop/lib/dashboard/src/ReactQueryDevtools.tsx

```diff
@@ -19,6 +20,11 @@ const ReactQueryDevtoolsProduction = React.lazy(() =>
  */
 export function ReactQueryDevtools() {
   const [showDevtools, setShowDevtools] = React.useState(false)
+  // It's safitier to pass the client directly to the devtools
+  // Since there might be a chance that we might have multiple versions if react-query
+  // in case if we forgot to update the devtools or if npm messed up with the versions
+  // or hoising issues.
```

Comment:
a couple typos in this comment. might want to pass this through grammarly or something